source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,142 @@
+.. _compass-create-vector-search:
+
+===================================
+Create an Atlas Vector Search Index
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.42.1
+
+You can create Atlas Vector Search indexes using |compass|. These indexes 
+enable you to index vector data *and* other data types, facilitating 
+semantic searches on the indexed fields. Atlas Vector Search indexes support 
+indexing vector data by identifying the most similar vectors. This is achieved 
+by calculating the distance between the query vector and the vectors stored 
+in the index.
```

Comment:
[suggestion to [avoid starting the sentence with "This"](https://www.mongodb.com/docs/meta/style-guide/writing/use-pronouns-carefully/#this)]

```suggestion
indexing vector data by identifying the most similar vectors. The index determines similarity 
by calculating the distance between the query vector and the vectors stored 
in the index.
```